virttest/virsh.py

```diff
@@ -2585,7 +2585,11 @@ def nodecpustats(option='', **dargs):
     """
 
     cmd_nodecpustat = "nodecpustats %s" % option
-    return command(cmd_nodecpustat, **dargs)
+    result = command(cmd_nodecpustat, **dargs)
```

Comment:
this is library call, check of the api behaviour has to be handled in individual testcase, in https://github.com/autotest/tp-libvirt/blob/master/libvirt/tests/src/virsh_cmd/host/virsh_nodecpustats.py